### PR TITLE
replace v1 UI with v2 UI

### PR DESCRIPTION
**Pull request by jacksonjacobs1**
This PR swaps in the V2 UI and adds cohortfinder and flask as requirements. The new UI developed incrementally on https://github.com/choosehappy/HistoQC/tree/ray_cluster_functionality 

Notes:
- The new flask-based UI requires the cohortfinder-choosehappy package, which is only currently available on the test pypi repository. See requirements.txt. 
- Although the cohortFinder API has not changed, the package is now out of date.
- CohortFinder has it's own requirements which are not yet automatically installed by pip.
- HistoQC's setup.py etc. do not yet reflect the changes in requirements.

Todos based on those notes:
- [x] Ensure that CohortFinder correctly installs it's dependencies
- [x] Upload the updated CohortFinder to PyPI
- [x] Include CohortFinder requirement in HistoQC's setup.py

Other todos:
- [ ] code cleanup: unused code, inconsistent comments, etc.

### CODE REVIEW

File: histoqc/ui/templates/abort.html [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
....svs', '121069.svs', 'TCGA-J2-8194-01Z-00-DX1.7700924D-B6AF-46A7-A7D7-B5C17A66C5F7.svs', '12748.svs', 'TCGA-E2-A158-01Z-00-DX1.994C60FE-E651-4224-95E7-4669834F2338.svs', '3511009759.svs', 'OCCCF25-4808.svs', '25038.svs', 'Slide-0001590_TI01747 4001.svs', 'TCGA-E9-A1RB-01Z-00-DX1.845074ea-43bc-4752-9dc0-fca72d2ab5de.svs', '54982.svs', 'TCGA-CH-5737-01Z-00-DX1.svs', 'Slide-0002181_TI05271 4002.svs', 'TCGA-A5-A0G5-DX1.svs', '12892.svs', 'Slide-0001473_TI01295 4006.svs', 'TCGA-LP-A5U3.svs', '54983.svs', 'TCGA-CI-6621-01A-01-BS1.d2846f26-bc31-489c-a76a-6dae8f36b293.svs', 'TCGA-BH-A0H5-01Z-00-DX1.28F24D4D-EE80-4EDA-BC30-A194E22FD61C.svs', '72012.svs', 'TCGA-AG-A00C-01A-02-BS2.05f8636d-8c83-49e0-ba22-2fe1db89b254.svs', '24624.svs', 'TCGA-AX-A06D-DX2.svs', 'TCGA-B5-A11R-DX1.svs', '1001639.svs', 'TCGA-DS-A1O9.svs', 'Slide-0001606_TI01752 4003.svs', '3369314.svs', '1001527.svs', '175775.svs', '9044.svs', 'TCGA-AY-A54L-01Z-00-DX1.BD4039B4-D732-418B-9CC9-064095A1F06F.svs', '175935.svs', 'TCGA-B5-A11P-DX1.svs', '15657.svs', 'S15-1274_PDL1.svs', 'Slide-0001870_TI04618 4004.svs', 'TCGA-EA-A3HS.svs', '118676.svs', '3311002523.svs', '1625960H.svs', 'ecce453f-c4d5-fd91-f836-14c938161269_123201.svs']
```
Comment (choosehappy) [inline histoqc/ui/templates/abort.html]: seems...strange?

File: histoqc/ui/templates/abort.html [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+        });
+      });
+
+      // controller.abort();
+    }
+
+    function runAnimation() {
+      progressAnim = setInterval(() => {
+        switch (animCount++ & 3) {
+          case 0: reports.textContent = 'Download occuring; waiting for video player to be constructed'; break;
```
Comment (choosehappy) [inline histoqc/ui/templates/abort.html]: ?

File: histoqc/ui/static/js/image_init.js [hunk 3]
```diff
@@ -260,6 +288,20 @@ function generateImgSrc(file_name, img_type_index, use_small = false) {
 	return window.location.origin + "/image/" + file_name + '/' + image_extension;
 }
 
+async function fetchImages(source, compare_source, signal) {
+	var imgURL = await fetch(source, { signal })
+		.then(response => response.blob())
+		.then(blob => URL.createObjectURL(blob));
+	var compareURL;
+
+	if (compare_source) {
```
Comment (choosehappy) [inline histoqc/ui/static/js/image_init.js]: sorry, its not obvious to me whats going on here with compare_source?

**Comment by jacksonjacobs1** [comment]
Ah, I probably should have mentioned that this was an intermediate commit. 

Just wanted to push some code I was playing with to continue working from my personal laptop.

**Comment by choosehappy** [comment]
oops : )

**Review by choosehappy — APPROVED** [review]

**Pull request merged**